slack_connector/main.py

```diff
@@ -51,14 +74,19 @@ def get_messages(api_token,channel_id):
 
             try:
 
-                result = client.conversations_history(channel=c)
+                result = client.conversations_history(channel = c)
                 conversation_history = result["messages"]
                 df_conversation_history = pd.DataFrame(conversation_history)
+
+                if df_conversation_history.empty == True:
+                    raise NoMessageError()
+
                 df.append(df_conversation_history)
             
             except SlackApiError as e:
             
                 logger.error("Error creating conversation list: {}".format(e))
```

Comment:
Is the  SlackApiError exception fatal? or just informational? 

if it is fatal, then  I think you should raise you own exception or re-raise the SlackApiError.